docs/learn.md

```diff
@@ -400,7 +400,318 @@ over the way these JavaScript packages are downloaded and bundled.
+  tests](https://dune.readthedocs.io/en/stable/tests.html), integration with
+  [Odoc](https://dune.readthedocs.io/en/stable/documentation.html), Melange,
+  [Js_of_ocaml](https://dune.readthedocs.io/en/stable/jsoo.html), [watch
+  mode](https://dune.readthedocs.io/en/stable/usage.html#watch-mode), Merlin/LSP
+  integration for editor support, [cross
+  compilation](https://dune.readthedocs.io/en/stable/cross-compilation.html),
+  and [generation of `opam`
+  files](https://dune.readthedocs.io/en/stable/opam.html#generating-opam-files).
+
+#### Creating a new project
+
+To understand how to use Dune, let’s create a small Melange application using
+it.
+
+First of all, create an opam switch, like shown in the [package management
+section](#package-management):
+
+```bash
+opam switch create . 4.14.1 --deps-only
+```
+
+Install Dune and Melange in the switch:
+
+```bash
+opam install dune melange
+```
+
+Create a file named `dune-project`. This file will tell Dune a few things about
+our project configuration:
+
+```bash
+(lang dune 3.8)
+
+(using melange 0.1)
+```
+
+The first line `(lang dune 3.8)` tells Dune which version of the "Dune language"
+(the language used in `dune` files) we want to use. Melange is only compatible
+with versions of the Dune language equal or greater than 3.8.
+
+The second line `(using melange 0.1)` tells Dune we want to use the [Melange
+extension of the Dune
+language](https://dune.readthedocs.io/en/stable/dune-files.html#using).
+
+#### Adding a library
+
+Next, create a folder `lib`, and a `dune` file inside. Put the following content
+inside the `dune` file:
+
+```bash
+(library
+ (name lib)
+ (modes melange))
+```
+
+Create a file `lib.ml` in the same folder:
+
+```ml
+let name = "Jane"
+```
+
+The top level configuration entries —like the `library` one that appears in the
+`dune` file— are referred to as _stanzas_, and the inner ones —like `name` and
+`modes`— are referred to as _fields_ of the stanza.
+
+All stanzas are well covered in the Dune documentation site, where we can find
+the reference for the [`library`
+stanza](https://dune.readthedocs.io/en/stable/dune-files.html#library).
+
+Dune is designed to minimize changes in configuration when the project folder
+structure changes, so one can move around the `lib` folder to another place
+inside the project, and all build commands will still keep working without any
+changes in Dune configuration. Very handy!
+
+#### Entry points with `melange.emit`
+
+Libraries are useful to encapsulate behavior and logical components of our
+application, but they won’t produce any JavaScript artifacts on their own.
+
+To generate JavaScript code, we need to define an entry point of our
+application. In the root folder, create another `dune` file:
+
+```bash
+(melange.emit
+ (target app)
+ (libraries lib))
+```
+
+And an `app.ml` file:
+
+```
+let () = Js.log Lib.name
+```
+
+The `melange.emit` stanza is the one that tells Dune to generate JavaScript
```

Comment:
"is the one that" is unnecessary in this sentence.

Reply:
https://github.com/melange-re/documentation-site/pull/25/commits/237a179eedbad549224da423dd5d26c3b0839181